lib/posthog/feature_flags.rb

```diff
@@ -92,9 +109,10 @@ def _load_feature_flags()
     end
 
     def _request(method, endpoint, use_personal_api_key = false, data = {})
-      uri = URI("https://#{@host}/#{endpoint}/?token=#{@project_api_key}")
+      uri = URI("https://#{@host}/#{endpoint}")
       req = nil
       if use_personal_api_key
+        uri = URI("https://#{@host}/#{endpoint}/?token=#{@project_api_key}")
```

Comment:
I think we always need a `project_api_key`, otherwise the call wouldn't work.

How we determine project_api_key token: https://github.com/PostHog/posthog/blob/master/posthog/api/utils.py#L104

So, your idea elsewhere of just `POST`ing the `api_key` is very sound. We should do it in both cases, and not put it in the request anymore

Reply:
I think we went over this and agreed this was fine for now?

Reply:
yep yep